command/operator_debug.go

```diff
@@ -251,9 +251,27 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 		}
 	}
 
-	// Resolve server prefixes
-	for _, id := range argNodes(serverIDs) {
-		c.serverIDs = append(c.serverIDs, id)
+	// Resolve servers
+	members, err := client.Agent().Members()
+	c.writeJSON(c.collectDir, "members.json", members, err)
+	// we always write the error to the file, but don't range if no members found
```

Comment:
```suggestion
	// We always write the error to the file, but don't range if no members found
```

Reply:
Fixed.  Also corrected a bug with the c.writeJSON line just prior.